src/LowercaseSentenceTokenizer.java

```diff
@@ -30,7 +31,32 @@ public class LowercaseSentenceTokenizer implements Tokenizer {
    */
   public List<String> tokenize(Scanner scanner) {
     // TODO: Implement this function to convert the scanner's input to a list of words and periods
-    return null;
+
+    // initialize list
+    List<String> inputStrings = new ArrayList<>();
+
+    // https://howtodoinjava.com/java/string/java-string-split-example/
+    // learned patterns for useDelimiter method here and applied using trial and error to find the right combo
+    // planned to use "useDelimiter", however I could not figure out the regex patterns for what I wanted, also learned it wouldn't be as effecient as what I have below
+    // scanner.useDelimiter("[,\\s]+|(?<=\\.)\\s+|(?=\\.)\\w+");
```

Comment:
Wow, that's quite a regex! Nice job both doing research, and recognizing when a simpler approach was warranted.